airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -654,21 +654,28 @@ class TicketMetrics(SourceZendeskSupportCursorPaginationStream):
     """TicketMetric stream: https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_metrics/"""
 
     def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
-        if self._ignore_pagination:
-            return None
-        next_page = self._parse_next_page_number(response)
```

Comment:
It looks like there is no more usages of the `_parse_next_page_number` method. Shouldn't we delete it as well?

Reply:
Not like really, other streams are  using it:
- All `SourceZendeskSupportFullRefreshStream` based streams are using it, due to different response structures for different streams.